features/typed-array-iterators.yml.dist

```diff
@@ -0,0 +1,56 @@
+# Generated from: typed-array-iterators.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2016-09-20
```

Comment:
The description really strongly implies we're treating those things as core to the feature. Either the description needs to change, or we need to bring `entries()` along for the ride. Feels more honest to do the latter, to change the status rather than reshape the feature.